oxidation/bindings/README.md

```diff
@@ -19,24 +33,134 @@ npm run build  # Generate index.node (basically a .so that node can load)
+     cmdline-tools;1.0    | 1.0          | Android SDK Command-line Tools
+     cmdline-tools;2.1    | 2.1          | Android SDK Command-line Tools
+     cmdline-tools;3.0    | 3.0          | Android SDK Command-line Tools
+     cmdline-tools;4.0    | 4.0          | Android SDK Command-line Tools
+     cmdline-tools;5.0    | 5.0          | Android SDK Command-line Tools
+     cmdline-tools;6.0    | 6.0          | Android SDK Command-line Tools
+     cmdline-tools;7.0    | 7.0          | Android SDK Command-line Tools
+     cmdline-tools;8.0    | 8.0          | Android SDK Command-line Tools
+     cmdline-tools;9.0    | 9.0          | Android SDK Command-line Tools
+     cmdline-tools;latest | 9.0          | Android SDK Command-line Tools (latest)
+   ```
+
+6. Install `cmdline-tools`
+
+   ```shell
+   $TEMP_DIR/cmdline-tools/bin/sdkmanager --install "cmdline-tools;latest"
+   ```
+
+   > Note: using `latest` may not allow reproductible build on different dev env.
+   > But you can replace the value by another one (`0.9` for example).
+
+7. Update your `PATH` variable
+
+   ```shell
+   export PATH="$PATH:$ANDROID_HOME/cmdline-tools/latest/bin"
+   ```
 
-- Install Android Studio
-- In `File > Settings > Appearance & Behavior > System Settings > Android SDK > SDK Platform > Show Package Details` : Install Android SDK 30.0.3
-- In `File > Settings > Appearance & Behavior > System Settings > Android SDK > SDK Tools > Show Package Details` : Install Android NDK 22.1.7171670 and Android SDK Command-line Tools
-- Python command should be available in PATH environment variable (used by `rust-android-gradle` plugin)
+   > Note: the path is `latest/bin` because we install `cmdline-tools` a the tag `latest`.
+   > For `9.0`, it will be `9.0/bin` for example.
+
+8. Install Android `SDK-30.0.3` and `NDK-22.1.7171670` by running
+
+    ```shell
+    sdkmanager --install "ndk;<ndk version>" "build-tools;<build tool version>"
+    ```
+
+#### Install required rust targets
 
 Install Rust targets for cross compilation:
 
-```bash
-rustup target add armv7-linux-androideabi
-rustup target add i686-linux-android
-rustup target add aarch64-linux-android
-rustup target add x86_64-linux-android
+```shell
+rustup target add \
+  armv7-linux-androideabi \
+  i686-linux-android \
+  aarch64-linux-android \
+  x86_64-linux-android \
+```
+
+> The `bindings/android` directory is a valid Android project to generate a libparsec AAR.
+> However the AAR doesn't have to be built when using `../client/android` project (given it depends explicitly on the `bindings/android/libparsec`).
+
+#### Install Java development kit to build the application
+
+Install the latest LTS of `openjdk`:
+
+- Install the latest java LTS development kit (currently `JDK-17`).
+- If need to be, update your `JAVA_HOME` to point to the installed jdk version folder.
+
+### Update the Gradle dependencies locks
+
```

Comment:
A small word about the difference between dependency lock and verification metadata would be useful here:
```
In most languages pinning dependency version and storing each dependency checksum is a single step (e.g. see `Cargo.lock`), but with Gradle this is divided into two steps:

- dependency locking  consists of pinning the list of dependencies (including transitive ones) use by the project
- verification metadata contains the list of dependency checksums
```

Reply:
done